_health-care/_js/_components/health-care-app.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import _ from 'lodash';
+import lodashDeep from 'lodash-deep';
+
+import IntroductionPanel from './introduction-panel.jsx';
+import Nav from './nav.jsx';
+
+// Add deep object manipulation routines to lodash.
+_.mixin(lodashDeep);
+
+class HealthCareApp extends React.Component {
+  constructor() {
+    super();
+    this.publishStateChange = this.publishStateChange.bind(this);
+    this.state = {
+      applicationData: {
+        introduction: {
+          happyDate: {
+            month: "1",
+            day: "15",
+            year: "1997",
+          },
+
+          sadDate: {
+            month: "4",
+            day: "5",
+            year: "2014",
+          }
+        },
+        personalInformation: {
```

Comment:
@awong-dev Please review this, I changed this in moving the `fullName` and `mothersMaidenName` components into the `personalInformationPanel`.